### ISSUE TITLE
Please fix the broken 3.8-1 release with wrong pom.xml 

### ISSUE THREAD

**Issue by mmokrejs** [broadinstitute/gatk#4685]
Hi,
  first of all, I find it very awkward that after 3.8 release there is 3.8-1. Why the dash instead of a dot, as usual? It only complicates automated package downloads which in general work with numbers separated by dots. You just mix together two schemes. Is that really necessary?

  Anyway, the pom.xml is broken:

```
>>> Preparing source in /scratch/var/tmp/portage/sci-biology/gatk-3.8.1/work/gatk-3.8-1 ...
Equivalent maven command
mvn -Dmaven.repo.local=/scratch/var/tmp/portage/sci-biology/gatk-3.8.1/work/.m2/repository verify '-Ddisable.shadepackage'
[INFO] Scanning for projects...
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[FATAL] Non-parseable POM /scratch/var/tmp/portage/sci-biology/gatk-3.8.1/work/gatk-3.8-1/public/gatk-root/pom.xml: unexpected character in markup < (position: END_TAG seen ...</artifactId>\n<<... @15:3)  @ /scratch/var/tmp/portage/sci-biology/gatk-3.8.1/work/gatk-3.8-1/public/gatk-root/pom.xml, line 15, column 3
 @ 
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]   
[ERROR]   The project org.broadinstitute.gatk:gatk-aggregator:[unknown-version] (/scratch/var/tmp/portage/sci-biology/gatk-3.8.1/work/gatk-3.8-1/pom.xml) has 1 error
[ERROR]     Non-parseable POM /scratch/var/tmp/portage/sci-biology/gatk-3.8.1/work/gatk-3.8-1/public/gatk-root/pom.xml: unexpected character in markup < (position: END_TAG seen ...</artifactId>\n<<... @15:3)  @ /scratch/var/tmp/portage/sci-biology/gatk-3.8.1/work/gatk-3.8-1/public/gatk-root/pom.xml, line 15, column 3 -> [Help 2]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
[ERROR] [Help 2] http://cwiki.apache.org/confluence/display/MAVEN/ModelParseException
```

```
  1 <?xml version="1.0" encoding="UTF-8"?>
  2 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
  3     <modelVersion>4.0.0</modelVersion>
  4 
  5     <!--
  6     This pom is parent for all gatk poms
  7     See also:
  8       http://maven.apache.org/pom.html#Inheritance_v
  9       http://maven.apache.org/guides/introduction/introduction-to-the-pom.html#Project_Inheritance_vs_Project_Aggregation
 10       http://stackoverflow.com/questions/1992213/maven-parent-pom-vs-modules-pom
 11     -->
 12 
 13     <groupId>org.broadinstitute.gatk</groupId>
 14     <artifactId>gatk-root</artifactId>
 15 <<<<<<< HEAD
 16     <version>3.8-1</version>
 17 =======
 18     <version>3.8-2-SNAPSHOT</version>
 19 >>>>>>> 0450e2531ee021e28bd7c5e92b5ba736d530d9af
 20     <packaging>pom</packaging>
 21     <name>GATK Root</name>
 22 
 23     <prerequisites>
 24         <maven>3.0.4</maven>
 25     </prerequisites>
```

Please make a new release, preferably 3.8.2 which unpacks as usual into `./gatk-3.8.2/`. Thank you.

While making a bugfix release? because it is the last version supporting `old` syntax.

**Comment by droazen** [comment]
@mmokrejs The 3.8-1 pom is not broken -- I just checked out and built it myself without error.

It looks to me like you are building from an unclean working tree with an unresolved merge conflict, since there are conflict markers in your copy of the pom:

```
 15 <<<<<<< HEAD
 16     <version>3.8-1</version>
 17 =======
 18     <version>3.8-2-SNAPSHOT</version>
 19 >>>>>>> 0450e2531ee021e28bd7c5e92b5ba736d530d9af
```

You should ensure that you have a clean checkout of `3.8-1` by running `git checkout -f 3.8-1` in your clone.

**Comment by mmokrejs** [comment]
But I used your tar.gz file from github. There are about 12 pom.xml  broken in the same way.

``
$ sha1sum -b /usr/portage/distfiles/gatk-3.8-1.tar.gz 
a47f211d1a34ccd52cf2b60a564e8cf348b9b618 */usr/portage/distfiles/gatk-3.8-1.tar.gz
```

**Comment by mmokrejs** [comment]
Actually I still have in xterm buffer the source:

```
>>> Downloading 'https://github.com/broadgsa/gatk/archive/3.8-1.tar.gz'
--2018-04-20 19:21:15--  https://github.com/broadgsa/gatk/archive/3.8-1.tar.gz
Resolving github.com... 192.30.253.112, 192.30.253.113
Connecting to github.com|192.30.253.112|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://codeload.github.com/broadgsa/gatk/tar.gz/3.8-1 [following]
--2018-04-20 19:21:16--  https://codeload.github.com/broadgsa/gatk/tar.gz/3.8-1
Resolving codeload.github.com... 192.30.253.121, 192.30.253.120
Connecting to codeload.github.com|192.30.253.121|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: unspecified [application/x-gzip]
Saving to: '/scratch/usr/portage/distfiles/gatk-3.8.1.tar.gz'

/scratch/usr/portage/distfiles/gatk-3.8.1.tar.gz          [                                   <=>                                                                                 ] 473.08M  1.07MB/s    in 8m 44s  

2018-04-20 19:30:01 (924 KB/s) - '/scratch/usr/portage/distfiles/gatk-3.8.1.tar.gz' saved [496061006]
```

**Comment by droazen** [comment]
@mmokrejs You need to actually `git clone` the gatk repo -- building directly from the `.tar.gz` is not supported.

**Comment by mmokrejs** [comment]
@droazen No way, the broken merge is comitted to the git tree, please check yourself. Probably all pom.xml files are affected.

**Comment by droazen** [comment]
@mmokrejs Had a look just now, and it appears that the `3.8-1` tag in the `broadgsa/gatk` repo is pointing at the wrong commit. It should be pointing at the latest commit on master (`41147a655594c2aae6e2cad8462bd1648570b32b`, which has the correct versions of the poms), and *not* at `1b984c10417b8d480a1d26ad9683437c0a45e77a`, which is indeed broken.

I'll ask someone to fix the tag, but for now you can build GATK 3.8-1 by cloning the repo, running `git checkout -f 41147a655594c2aae6e2cad8462bd1648570b32b`, and then building as usual. Once the tag is fixed to point at the right commit, you can change the checkout command to `git checkout -f 3.8-1` for future builds.

Thanks for the report!

**Comment by droazen** [comment]
ping @vdauwera

**Comment by droazen** [comment]
@vdauwera monthly ping :)

**Comment by lbergelson** [comment]
@vdauwera Ping

**Comment by droazen** [comment]
We've fixed the `3.8-1` tag in the `broadgsa/gatk` repo -- closing!
